client/landing/gutenboarding/lib/analytics/index.ts

```diff
@@ -38,12 +39,16 @@ export function recordOnboardingStart( ref = '' ): void {
  * Analytics call at the completion  of a Gutenboarding flow
  *
  * @param {object} params A set of params to pass to analytics for signup completion
- * @param {boolean} params.isNewUser Whether the user is newly signed up
- * @param {boolean} params.isNewSite Whether a new site is created in the flow
  */
-export function recordOnboardingComplete( { isNewUser = false, isNewSite = true } ): void {
-	trackEventWithFlow( 'calypso_signup_complete', {
-		is_new_user: isNewUser,
-		is_new_site: isNewSite,
-	} );
+export function recordOnboardingComplete( params: OnboardingCompleteParameters ): void {
+	trackEventWithFlow( 'calypso_signup_complete', params );
+}
```

Comment:
These event params were using snake_case, and now they're using camelCase. I think snake case is correct because it matches what the `/start` uses.

The old flow also has a `has_cart_items` param, which maybe we should also add in another PR 🤔 it'd be an easy way to compare which flow is better at selling during onboarding.

Reply:
Argh, thanks for reminding me. I had a commit ready to fix this very thing last night.

...

Didn't push. :)